src/app/static/src/js/components/runWorkflow/workflowSummary/workflowSummaryReports.vue

```diff
@@ -22,7 +22,26 @@
                                     <div class="workflow-summary-text">
                                         <span class="text-muted d-inline-block">Parameters</span>
                                         <div v-if="hasParams(report)">
-                                            <p id="params" v-for="(value, key, index) in report.params" :key="index">{{ key }}: {{ value }}</p>
+                                            <p id="params"
+                                               v-for="(value, key, index) in report.params"
+                                               :key="index">{{ key }}: {{ value }}</p>
```

Comment:
How does this exclude the params which are defaults? They should only be visible when collapsible part is expanded.